client/src/Components/TaskEstimatedBox/TaskEstimatedBox.jsx

```diff
@@ -9,36 +9,38 @@ import CloseIcon from '@material-ui/icons/Close';
 import { useSocket } from 'socketio-hooks';
 
 const TaskEstimatedBox = () => {
+  const classes = TaskEstimatedBoxStyles();
+
   const [open, setOpen] = useState(false);
-  const [taskName, setTaskName] = useState();
-  const [userVoted, setUserVoted] = useState([]);
-  const [resultsAnalysis, setresultsAnalysis] = useState({});
+  const [taskName, setTaskName] = useState({});
```

Comment:
nazwałbym to task i pominał inicjalizację (ewentualnie podał null w środku) inicjalizacja pustym obiektem wg mnie nie ma sensu w tym przypadku